src/dynamo/programs/dynamo2xyz

```diff
@@ -39,21 +39,21 @@ for filename in sys.argv[1:]:
 
     #Now check that all files have the same particle count
     if particlecount != currentparticlecount:
-        print "input file", filename, "has",currentparticlecount,"particles, but all files must have the same number of particles. The first file has",particlecount
+        print("input file", filename, "has",currentparticlecount,"particles, but all files must have the same number of particles. The first file has",particlecount)
         exit(1)
         
     #Start of the XYZ file format. We skip the optional comment line
     #line 1: number of particles
-    print particlecount
+    print(particlecount)
     #line 2: molecule name
-    print "DynamOdata"
+    print("DynamOdata")
     #line 3-onwards: atom_number x y z vx vy vz
     for particleNode in RootElement.findall('.//Pt'):
         posNode = particleNode.find('P')
         velNode = particleNode.find('V')
         #First column is the atom name/type. Note the trailing comma to prevent a newline!
-        print "H", 
+        print("H",end=" ")
         #Next three columns are the position
-        print posNode.attrib['x'], posNode.attrib['y'], posNode.attrib['z'],
+        print(posNode.attrib['x'], posNode.attrib['y'], posNode.attrib['z'], end=" ")
         #Next three columns are the velocity
-        print velNode.attrib['x'], velNode.attrib['y'], velNode.attrib['z']
+        print(velNode.attrib['x'], velNode.attrib['y'], velNode.attrib['z'])
```

Comment:
Another useful change thanks!

Reply:
Thanks. I changed this as the rest of my workflow uses python 3. I think there some tools that automate a lot of the process of moving to 3, but I haven't tried them, so I limited myself to changing print statements to being forward compatible. I don't plan on doing anything else with python for the moment.

Reply:
Ok no worries, I'm entirely python3 now but old scripts in DynamO are still 2.7.